static/elements/chromedash-featurelist.html

```diff
@@ -152,18 +164,41 @@
 
       // Returns the closest feature <li> to the scroll top position passed in.
       featureInView: function(containerScrollTop) {
-        var closest = null;
-        for (var i = 0, li; li = featureLiList_[i]; ++i) {
-          var dist = li.offsetTop - containerScrollTop;
-          if (dist < 0) {
-            dist = -dist * 1.5;
-          }
-          if (closest == null || dist < closest.dist) {
-            closest = this.filtered[i];
-            closest.dist = dist;
-          }
+
+        // Generate list of features below current scroll position.
+        var items = featureLiList_.filter(function(el) {
+          if (el.offsetTop < containerScrollTop) { return el; }
```

Comment:
scroll events are async in chrome. If it becomes an actual bottleneck, then we can consider it.